client/service/src/task_manager/mod.rs

```diff
@@ -41,6 +41,26 @@ mod tests;
 /// Default task group name.
 pub const DEFAULT_GROUP_NAME: &'static str = "default";
 
+pub enum GroupName {
+	Default,
+	Specific(&'static str),
```

Comment:
And maybe something better than `Specific`? I mean this was just an example of me, with a little bit bad naming. However, I currently don't have any better idea.

Reply:
Oh... I see @bkchr 
But don't really know what to rename it to, how about `Actual`? like actual group name
I think `Specific` is still good too

Reply:
@bkchr I renamed Specific to Actual. Does that suffice?

Reply:
@ordian suggested we use Specific, so i am going ahead with that since it is what he prefers